### DISCUSSION TITLE
Couldn't get the debugging working

### DISCUSSION THREAD

**Discussion by kubikap** [evershopcommerce/evershop#561] [Q&A]
I keep getting the unbound breakpoint in VSCODE. I may be doing something wrong in launch.json.. Could you please advise? Thanks in advance.

**Comment by treoden** [comment]
It happens with EverShop only?

**Reply by kubikap** [reply]
Yes. Here is my launch.json. My assumption is that i am putting in wrong webroot. Please advise.

{
    // Use IntelliSense to learn about possible attributes.
    // Hover to view descriptions of existing attributes.
    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
    "version": "0.2.0",
    "configurations": [
        {
            "type": "chrome",
            "request": "launch",
            "name": "Launch Chrome against localhost",
            "url": "http://localhost:3000",
            "webRoot": "${workspaceFolder}"
        }
    ]
}
